pkg/server/sandbox_status.go

```diff
@@ -56,6 +58,8 @@ func (c *criContainerdService) PodSandboxStatus(ctx context.Context, r *runtime.
 		if taskStatus.Status == containerd.Running {
 			state = runtime.PodSandboxState_SANDBOX_READY
```

Comment:
Is it better that putting as below into toCRISandboxStatus?
We get the original info at the outside, processing it inside.
```
if taskStatus.Status == containerd.Running {
 			state = runtime.PodSandboxState_SANDBOX_READY
```

Reply:
Will send another PR to do that. Thanks!